test/unit-test/test_lib/test_analysis/test_network.py

```diff
@@ -71,34 +74,40 @@ def test_message_number_graph(self, mock_get_nick_sen_rec, mock_rec_list_splice,
     @mock.patch('lib.util.correctLastCharCR', autospec=True)
     @mock.patch('lib.util.rec_list_splice', autospec=True)
     @mock.patch('lib.util.get_nick_sen_rec', autospec=True)
-    def test_message_number_graph_day_analysis(self, mock_get_nick_sen_rec, mock_rec_list_splice, mock_correctLastCharCR, \
-                                            mock_check_if_msg_line, mock_create_connected_nick_list, mock_to_graph):
-        to_graph_ret = util.load_from_disk(current_directory + "/data/message_number_graph/to_graph")
-        
+    def test_message_number_graph_day_analysis(self, mock_get_nick_sen_rec, mock_rec_list_splice,
+                                               mock_correctLastCharCR, \
```

Comment:
why do we need a backslash character here? It might be better to put one more parameter on this line to balance the ends of all the three lines.

Reply:
Due to Pep 8 recommendation, this was done automatically by the IDE. I'll fix this